recipe/conda_build_config.yaml

```diff
@@ -4,42 +4,47 @@ c_stdlib_version:  # [osx and x86_64]
-  - 11                                              # [linux and aarch64]
-  - 12                                              # [linux and aarch64]
-fortran_compiler_version:                           # [linux and aarch64]
-  - 13                                              # [linux and aarch64]
-  - 11                                              # [linux and aarch64]
-  - 12                                              # [linux and aarch64]
-cdt_name:                                           # [linux and aarch64]
-  - cos7                                            # [linux and aarch64]
-  - cos7                                            # [linux and aarch64]
-  - cos7                                            # [linux and aarch64]
-cuda_compiler:                                      # [linux and aarch64]
-  - None                                            # [linux and aarch64]
-  - nvcc                                            # [linux and aarch64]
-  - cuda-nvcc                                       # [linux and aarch64]
-cuda_compiler_version:                              # [linux and aarch64]
-  - None                                            # [linux and aarch64]
-  - 11.8                                            # [linux and aarch64]
-  - 12.0                                            # [linux and aarch64]
+c_compiler_version:         # [osx]
+  - 17                      # [osx]
+cxx_compiler_version:       # [osx]
+  - 17                      # [osx]
+llvm_openmp:                # [osx]
+  - 17                      # [osx]
+
+mkl:
+  - 2023
+libprotobuf:
+  - 5.28.2
+
+# override global pinning to stay on CUDA 12.0
+docker_image:                                       # [linux]
+  - quay.io/condaforge/linux-anvil-x86_64:alma9     # [linux]
+  - quay.io/condaforge/linux-anvil-x86_64-cuda11.8:ubi8  # [linux]
+  - quay.io/condaforge/linux-anvil-x86_64:alma9     # [linux]
+c_compiler_version:                                 # [linux]
+  - 13                                              # [linux]
+  - 11                                              # [linux]
+  - 12                                              # [linux]
+cxx_compiler_version:                               # [linux]
+  - 13                                              # [linux]
+  - 11                                              # [linux]
+  - 12                                              # [linux]
+fortran_compiler_version:                           # [linux]
+  - 13                                              # [linux]
+  - 11                                              # [linux]
+  - 12                                              # [linux]
+cdt_name:                                           # [linux]
+  - cos7                                            # [linux]
+  - cos7                                            # [linux]
+  - cos7                                            # [linux]
```

Comment:
Forgot to remove this one with all the recent zip changes... Oh well, not important enough to repush